site/static/main.ts

```diff
@@ -2,6 +2,9 @@
 
 declare const BASEURL, hljs;
 
+// IIFE to prevent function declarations from moving into the global scope
```

Comment:
Thank you for the explanation. I think a separate `tsconfig` would be the clean solution but to reduce complexity, let's stick with one until it becomes a problem.